### DISCUSSION TITLE
Browser Extension: Selected entry in Drop-Down available without Auto-Type

### DISCUSSION THREAD

**Discussion by macpo** [keepassxreboot/keepassxc#10434] [Q&A]
Hi,

in general I want the browser extension to use fill out automatically (Auto-Type) the user name, password and if available the OTP. But I want to deactivate Auto-Type for selected sites like the Synology DSM or Portainer in a way that I need to select manually the entry in the login site and Auto-Type is deactivated for this entry.
Background: If I add or change a user the password is filled by the browser extension even after I set the password.

I activated Auto-Type in KeePassXC and in the browser extension the options "Fill out entries with single login data automatically" and I deactivated the setting "Activate Auto-Type for this entry" in KeePassXC for the Synology DSM. But the entry is still used to fill out the user and the password.

How can I deactivate Auto-Type only for selected entries?

**Comment by droidmonkey** [comment]
Auto-Type and Browser Integration are completely different features.

**Comment by macpo** [comment]
I created a new ticket for the browser extension: https://github.com/keepassxreboot/keepassxc-browser/issues/2157
Therefore this ticket is closed.
